Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM node:14.16.0-buster-slim
```

Comment:
Would it be worth switching to the active LTS node release (16)?

Reply:
yes